build-master.sh

```diff
@@ -3,10 +3,8 @@
 # build
 REPONAME="ats-ingress"
 TAG="latest"
-#TARGET="quay.io"/${QUAY_USERNAME}/${REPONAME}:${TAG}
-TARGET="ats-ingress:latest"
-#echo ${TARGET}
+TARGET=atsingress/${REPONAME}:${TAG}
```

Comment:
change it back to ats-ingress instead ?

Reply:
the final target has to be atsingress/ats-ingress:latest
atsingress is the organization name on docker hub (which i created) while ats-ingress is the repo name. org name can only have letters and numbers on dockerhub